docker/Dockerfile.package-cpu

```diff
@@ -27,17 +27,16 @@ RUN bash /install/centos_install_arm_ethosn_driver_stack.sh
 COPY install/centos_install_arm_compute_library.sh /install/centos_install_arm_compute_library.sh
 RUN bash /install/centos_install_arm_compute_library.sh
 
+# Install Conda
```

Comment:
just curious, why this change?

Reply:
[centos_install_python_packages](https://github.com/tlc-pack/tlcpack/blob/9928a5d01ca4b692645953917af25a95ab13b92c/docker/install/centos_install_python_package.sh#L18) will install the required python packages to run the tests for the current Conda environment. We need to first install Conda before doing that.